tests/integration/bug97_varmap_leak.rs

```diff
@@ -32,6 +32,7 @@ fn bug97_varmap_leak() {
         "origExtraClosure",
         // Nix-set variables
         "IN_NIX_SHELL",
+        "IN_LORRI_SHELL",
```

Comment:
Nit: this is listed under "Nix-set variables" although it is really set by lorri. I suggest creating a new "section" here, perhaps "lorri-set variables".

Reply:
I noticed this too haha. I'll add another section along with the integration test.